sgv2-docsapi/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Should this have "io.stargate:stargate" as its parent? (to get various properties, some managed dependencies)

Reply:
Mentioned in the PR description that is not yet connected to the parent..

Reply:
Ah. Yes, makes sense due to tooling challenges etc.